js/ui/scheduler/appointmentPopup.js

```diff
@@ -194,15 +194,15 @@ export default class AppointmentPopup {
         this.state.appointment.isEmptyText = data === undefined || data.text === undefined;
         this.state.appointment.isEmptyDescription = data === undefined || data.description === undefined;
 
-        const formData = extend({}, { text: '', description: '', recurrenceRule: '' }, this._createAppointmentFormData(data));
+        const formData = extend({ text: '', description: '', recurrenceRule: '' }, this._createAppointmentFormData(data));
 
         if(processTimeZone) {
             if(startDate) {
```

Comment:
In my opinion, the need for such verification is in doubt. Can we refactor it in this PR?

Reply:
No problem